server/controllers/shared/caseListController.ts

```diff
@@ -1,27 +1,42 @@
 import type { Request, Response, TypedRequestHandler } from 'express'
+import createError from 'http-errors'
 
-import type { ReferralService } from '../../services'
-import { ReferralUtils, TypeUtils } from '../../utils'
+import type { CourseService, ReferralService } from '../../services'
+import { CourseUtils, ReferralUtils, StringUtils, TypeUtils } from '../../utils'
 
 export default class CaseListController {
-  constructor(private readonly referralService: ReferralService) {}
+  constructor(
+    private readonly courseService: CourseService,
+    private readonly referralService: ReferralService,
+  ) {}
 
   show(): TypedRequestHandler<Request, Response> {
     return async (req: Request, res: Response) => {
       TypeUtils.assertHasUser(req)
 
+      const { courseName } = req.params
       const { audience, status } = req.query as Record<string, string>
-
       const { activeCaseLoadId, username } = res.locals.user
 
+      const courses = await this.courseService.getCoursesForOrganisation(username, activeCaseLoadId)
```

Comment:
I think we might want to unique this based on name, I've just tested this on Dev and Bure has two possible strands and shows duplicate courses in the tab.

<img width="1283" alt="image" src="https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/assets/19826940/7c93b5ac-c7ba-4080-8477-bc2a311c12cd">